README.md

```diff
@@ -181,218 +181,7 @@ transparently:
-[useless_format](https://github.com/rust-lang-nursery/rust-clippy/wiki#useless_format)                                       | warn    | useless use of `format!`
-[useless_let_if_seq](https://github.com/rust-lang-nursery/rust-clippy/wiki#useless_let_if_seq)                               | warn    | unidiomatic `let mut` declaration followed by initialization in `if`
-[useless_transmute](https://github.com/rust-lang-nursery/rust-clippy/wiki#useless_transmute)                                 | warn    | transmutes that have the same to and from types or could be a cast/coercion
-[useless_vec](https://github.com/rust-lang-nursery/rust-clippy/wiki#useless_vec)                                             | warn    | useless `vec!`
-[verbose_bit_mask](https://github.com/rust-lang-nursery/rust-clippy/wiki#verbose_bit_mask)                                   | warn    | expressions where a bit mask is less readable than the corresponding method call
-[while_let_loop](https://github.com/rust-lang-nursery/rust-clippy/wiki#while_let_loop)                                       | warn    | `loop { if let { ... } else break }`, which can be written as a `while let` loop
-[while_let_on_iterator](https://github.com/rust-lang-nursery/rust-clippy/wiki#while_let_on_iterator)                         | warn    | using a while-let loop instead of a for loop on an iterator
-[wrong_pub_self_convention](https://github.com/rust-lang-nursery/rust-clippy/wiki#wrong_pub_self_convention)                 | allow   | defining a public method named with an established prefix (like "into_") that takes `self` with the wrong convention
-[wrong_self_convention](https://github.com/rust-lang-nursery/rust-clippy/wiki#wrong_self_convention)                         | warn    | defining a method named with an established prefix (like "into_") that takes `self` with the wrong convention
-[wrong_transmute](https://github.com/rust-lang-nursery/rust-clippy/wiki#wrong_transmute)                                     | warn    | transmutes that are confusing at best, undefined behaviour at worst and always useless
-[zero_divided_by_zero](https://github.com/rust-lang-nursery/rust-clippy/wiki#zero_divided_by_zero)                           | warn    | usage of `0.0 / 0.0` to obtain NaN instead of std::f32::NaN or std::f64::NaN
-[zero_prefixed_literal](https://github.com/rust-lang-nursery/rust-clippy/wiki#zero_prefixed_literal)                         | warn    | integer literals starting with `0`
-[zero_ptr](https://github.com/rust-lang-nursery/rust-clippy/wiki#zero_ptr)                                                   | warn    | using 0 as *{const, mut} T
-[zero_width_space](https://github.com/rust-lang-nursery/rust-clippy/wiki#zero_width_space)                                   | deny    | using a zero-width space in a string literal, which is confusing
+https://rust-lang-nursery.github.io/rust-clippy/master/index.html
```

Comment:
I suggest something like

```md
[There are 209 lints included in this crate!](https://rust-lang-nursery.github.io/rust-clippy/master/index.html)
```

which renders as

> [There are 209 lints included in this crate!](https://rust-lang-nursery.github.io/rust-clippy/master/index.html)

Rather that the current which renders as

> There are 209 lints included in this crate: https://rust-lang-nursery.github.io/rust-clippy/master/index.html